public/app/plugins/datasource/graphite/datasource.test.ts

```diff
@@ -503,3 +508,22 @@ accessScenario('with direct access', 'http://localhost:8080', (httpOptions: any)
   expect(httpOptions.headers['X-Dashboard-Id']).toBe(undefined);
   expect(httpOptions.headers['X-Panel-Id']).toBe(undefined);
 });
+
+const createFetchResponse = (response: Partial<FetchResponse>): Observable<FetchResponse> => {
```

Comment:
I think we already have a shared function for this in `public/test/helpers/createFetchResponse.ts`.